helm-chart/slack-mcp-client/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.networkPolicy.enabled }}
```

Comment:
I'm curious what is a use case for kubernetes service in this case?

Reply:
My bad the service is not needed and has no use case for now. I removed it.